toolkit/block-production-log/primitives/src/lib.rs

```diff
@@ -62,20 +63,23 @@ use {
 	sp_inherents::InherentData,
 };
 
+/// Inherent identifier used by the Block Production Log pallet
 pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"blprdlog";
 
 /// Error type used for failing calls of the block production log feature's inherent.
 #[derive(Encode, PartialEq)]
 #[cfg_attr(not(feature = "std"), derive(Debug))]
 #[cfg_attr(feature = "std", derive(Decode, thiserror::Error, sp_runtime::RuntimeDebug))]
 pub enum InherentError {
+	/// Inherent was not producer when expected
 	#[cfg_attr(
 		feature = "std",
 		error("Block Author inherent must be provided every block after initialization")
 	)]
 	InherentRequired,
+	/// The inherent was produced with incorrect production data
```

Comment:
I don't understand the "production" part. This is used when deserialization of inherent data fails, like the message below points. IMO it is enough to c-p the message below to satisfy rules checker.